addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/utils/Element.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2020 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.openapi.utils;
+
+public final class Element {
+
+    private Element() {}
+
+    public static class Json {
```

Comment:
Should be an Enum (same for following classes).

Reply:
Inner enums are not looking great as I will need a dummy value like this:

`public enum Element {

    ELEMENT;

    public enum Form {
        INNER_SEPARATOR(":"),
        OUTER_SEPARATOR(",");

        private String syntax;
`
What do you think?

Reply:
The `Element` can be kept as class.